charts/internal/cri-resmgr-installation/templates/installaton-configmap.yaml

```diff
@@ -195,7 +195,7 @@ data:
           active_policy=$(chroot /var/host bash -c 'cat /var/lib/cri-resmgr/cache | jq .PolicyName -r')
           cfg_data=$(chroot /var/host bash -c 'cat /var/lib/cri-resmgr/cache | jq .Cfg.Data -r')
           if [[ "$cfg_data" == "null"  ]]; then
-              requested_policy="null"
+              requested_policy="none"
```

Comment:
good catch! I've just found the commit responsible for that: https://github.com/intel/cri-resource-manager/commit/b2969c25d966b6ad8d61a64216df4bfc5c5fdf5d

Now it is definitly better that we don't need to distingish between null and none policy.